server/services/personService.ts

```diff
@@ -31,4 +32,17 @@ export default class PersonService {
     const rosh = await personClient.oasysRosh(crn)
     return rosh
   }
+
+  async getRoshRisks(token: string, crn: string): Promise<RoshRisks> {
+    const personClient = this.personClientFactory(token)
+
+    const risks = await personClient.risks(crn)
+
+    return {
+      ...risks.roshRisks?.value,
```

Comment:
I think there's at least one unhappy path not accounted for (and not covered by unit tests on lines 42-43.)

When I try to run this locally the data I get back from the API does not have `roshRisks` and so `lastUpdated` is '', which is causing an `Invalid Date` error on the UI.